pkg/controllers/ingressnodesavailable/ingress_nodes_available_controller.go

```diff
@@ -51,7 +51,7 @@ func NewIngressNodesAvailableController(
 			nodeInformer.Informer(),
 		).
 		WithSync(controller.sync).
-		ResyncEvery(1*time.Minute).
+		ResyncEvery(time.Minute).
```

Comment:
would it make sense to provide some jitter? so that not all controllers try to reach the API at the same time?

Reply:
Done.